File: Dart/bubble_sort.dart
void main() {
  List<int> arr = [60, 31, 29, 19, 33, 9, 111];
  bubbleSort(arr);
  print("Sorted array");
  print(arr);
}

void bubbleSort(List<int> arr) {
  int n = arr.length;
  for (int i = 0; i < n - 1; i++) {
    for (int j = 0; j < n - i - 1; j++)
      if (arr[j] > arr[j + 1]) {
        int temp = arr[j];
        arr[j] = arr[j + 1];
        arr[j + 1] = temp;
      }
  }
}

File: C Programming/bubblesort.c
#include <stdio.h>

int main(){

    int arr[50], num, x, y, temp;    

    printf("Please Enter the Number of Elements you want in the array: ");

    scanf("%d", &num);    

    printf("Please Enter the Value of Elements: ");

    for(i = 0; i < num; i++)

        scanf("%d", &arr[i]);

    for(i = 0; i < num - 1; i++){       

        for(y = 0; y < num - i - 1; y++){          

            if(arr[y] > arr[y + 1]){               

                temp = arr[y];

                arr[y] = arr[y + 1];

                arr[y + 1] = temp;

            }

        }

    }

    printf("After implementing bubble sort: ");

    for(i = 0; i < num; i++){

        printf("%d  ", arr[i]);

    }

    return 0;

}
